monai/transforms/lazy/functional.py

```diff
@@ -25,14 +25,33 @@
 )
 from monai.utils import LazyAttr, look_up_option
 
-__all__ = ["apply_transforms"]
+__all__ = ["apply_pending", "execute_pending_transforms"]
 
 __override_keywords = {"mode", "padding_mode", "dtype", "align_corners", "resample_mode", "device"}
 
 
-def apply_transforms(
-    data: torch.Tensor | MetaTensor, pending: list | None = None, overrides: dict | None = None, **kwargs: Any
-):
+def execute_pending_transforms(data, overrides: dict | None = None):
```

Comment:
please add docstring, also `execute_pending_transforms` and `apply_pending` have very similar name and signature, it's unclear what to use without reading the source code

Reply:
Will do

Reply:
@wyli done